code/game/objects/items/stacks/sheets/sheets.dm

```diff
@@ -39,6 +39,8 @@
 			var/obj/item/new_shard = new mat.shard_type(user.loc)
 			new_shard.add_fingerprint(user)
 			shards += "\a [new_shard.name]"
+			if(mat.debris_type)
+				new mat.debris_type(user.loc)
```

Comment:
man we use `user.loc` twice in this for loop i think we could probably cache it beforehand so we don't have to keep doing fuckshit work for when people are sharding up the place